tests/framework/utils_cpuid.py

```diff
@@ -130,3 +130,27 @@ def get_guest_cpuid(vm):
     assert stderr.read() == ""
 
     return build_cpuid_dict(stdout)
+
+
+def check_cpuid_feat_flags(vm, must_be_set, must_be_unset):
```

Comment:
nit: I appreciate it might not be worth it, but moving this to the utils is refactoring and could have been placed in a separate commit.